pkg/tsdb/sqleng/resample.go

```diff
@@ -0,0 +1,128 @@
+package sqleng
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/grafana/grafana-plugin-sdk-go/data"
+)
+
+func getRowFillValues(f *data.Frame, tsSchema data.TimeSeriesSchema, currentTime time.Time,
```

Comment:
I wonder if we are replicating behavior already done in the sdk, since LongToWide takes fillmissing 

https://github.com/grafana/grafana-plugin-sdk-go/blob/867dddf55bd1438063435f15a1638f1bf6c2d605/data/time_series.go#L206

https://github.com/grafana/grafana-plugin-sdk-go/blob/master/data/time_series.go#L290

Reply:
Oh I think I see, this some additional post processing for some reason :-) :+1: